code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -27,9 +27,10 @@
 	..()
 
 
-/obj/item/robot_module/New()
+/obj/item/robot_module/New(loc, should_have_flash = TRUE)
 	..()
-	modules += new /obj/item/flash/cyborg(src)
+	if(should_have_flash)
+		modules += new /obj/item/flash/cyborg(src)

```

Comment:
Wait no what was I thinking
Adding extra items to a module should be put in a separate proc, and then override that proc in the drone not to give the item you don't want them to have

Reply:
I feel like I should just go ahead and do that for every module, medical, mining, sec etc. Just throw every module's item giving code into some proc, name it `add_module_specific_items` or something, then call that in `New()`